_i18n/fr/get-started/faq.md

```diff
@@ -71,7 +71,7 @@ Pour un portefeuille léger, vous communiquer votre clef publique d'audit à un
 
 <div class="tab-content" markdown="1">
 
-Monero n'est pas basé sur Bitcoin. Il est basé sur le protocole CryptoNote. Bitcoin est un système totalement transparent, où chacun peut voir quel est le montant exacte d'une transaction envoyé d'un utilisateur à un autre. Monero masque ces informations afin de protéger la confidentialité de ses utilisateurs dans toutes les transactions. Il dispose également d'une taille de bloc et de frais de transferts dynamiques, d'une preuve de travail (proof of work) résistant aux ASICs, d'une émission de queue de pièces de monnaies, ainsi que d'autres changements.
+Monero n'est pas basé sur Bitcoin. Il est basé sur le protocole CryptoNote. Bitcoin est un système totalement transparent, où chacun peut voir quel est le montant exact d'une transaction envoyé d'un utilisateur à un autre. Monero masque ces informations afin de protéger la confidentialité de ses utilisateurs dans toutes les transactions. Il dispose également d'une taille de bloc et de frais de transferts dynamiques, d'une preuve de travail (proof of work) résistant aux ASICs, d'une émission de queue de pièces de monnaies, ainsi que d'autres changements.
```

Comment:
où chacun peut voir quel est le montant exact d'une transaction envoyé d'un utilisateur à un autre -> où chacun peut voir le montant exact d'une transaction envoyée d'un utilisateur à un autre

Reply:
ok for both.
"émission résiduelle perpétuelle" is fine to me, but perhpaps "émission résiduelle" would be enough? As long as we don't says it will stop, it is perpetual.

Reply:
So i'll go with "courbe résiduelle" line 82 of tech-specs.

Reply:
Sounds good.